specification/riskiq/Easm/main.tsp

```diff
@@ -0,0 +1,51 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/versioning";
+import "./routes.tsp";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using TypeSpec.Versioning;
+
+@useAuth(OAuth2Auth<[{
+        @doc("implicit flow")
+        type: OAuth2FlowType.implicit;
+
+        @doc("the authorization URL")
+        authorizationUrl: "https://login.microsoftonline.com/common/oauth2/authorize";
+
+        @doc("list of scopes for the credential")
+        scopes: ["https://easm.defender.microsoft.com/.default"];
+    }]>)
+@versioned(Easm.Versions)
+@service({
+    title: "Defender EASM",
+})
+@server(
+    "{endpoint}/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/workspaces/{workspaceName}",
```

Comment:
Why is the server url defined like this? You most likely just want `{endpoint}` like it was defined in the original swagger.

Reply:
Nevermind, I just noticed what the routes look like, it seems better to specify these parameters once at the server level instead of on each operation